doc/api/cli.md

```diff
@@ -620,6 +620,13 @@ Enables report to be generated on un-caught exceptions, if
 `--experimental-report` is enabled. Useful when inspecting JavaScript stack in
 conjunction with native stack and other runtime environment data.
 
+### `--sigint-stacktrace`
```

Comment:
A little naming suggestion: `--trace-sigint` to keep conventions at https://github.com/nodejs/node/blob/b43895cc3c8f05ea7dabc8dd1f222b73da107d4f/doc/api/cli.md#--trace-deprecation.

Reply:
Sounds good to me.